nodl_python/nodl/errors.py

```diff
@@ -117,3 +125,6 @@ class UnsupportedInterfaceError(InvalidNoDLError):
 
     def __init__(self, version: int, max_version: int) -> None:
         super().__init__(f'Unsupported interface version: {version} must be <= {max_version}')
+
+
+# endregion
```

Comment:
Is this some vscode thing?

Reply:
Yes, I was cherry picking from the cli branch to make this and I carelessly left this in, removed in 97698c7